tests/helpers/faux-context.js

```diff
@@ -1,33 +0,0 @@
-'use strict';
```

Comment:
Was this file used?

Reply:
I have not found where it is used and the tests work without it.